docs/users-guide/custom-questions.md

```diff
@@ -97,14 +97,20 @@ Here are the basic types of joins:
 
 **A left outer join example:** If Table A is Orders and Table B is Customers, and assuming you do a join where the `customer_id` column in Orders is equal to the `ID` column in Customers, when you do a left outer join your results will be a full list of all your orders, and each order row will also display the columns of the customer who placed that order. Since a single customer can place many orders, a given customer's information might be repeated many times for different order rows. If there isn't a corresponding customer for a given order, the order's information will be shown, but the customer columns will just be blank for that row.
 
-##### Multiple stages of joins
+### Multiple stages of joins
 
 In many cases you might have tables A, B, and C, where A and B have a connection, and B and C have a connection, but A and C don't. If you want to join A to B to C, all you have to do is add multiple join steps. Click on Join Data, join table A to table B, then click the Join Data step below that completed join block to add a second join step, and join the results of your last join to table C.
 
 ![An A to B to C join](./images/notebook/join-a-b-c.png)
 
 See [Joins in Metabase](https://www.metabase.com/blog/joining-tables/index.html) to learn more.
 
+### Joining on multiple columns
+
+You can also join data on multiple columns to further refine your results.
```

Comment:
"You can also join data on multiple columns" — maybe more precisely, "Your joins can now include multiple conditions…"

Might want to call out that these conditionals are `AND`ed together for now.